style-guidelines/Swift.md

```diff
@@ -141,8 +141,14 @@ else {
 
 ## Classes and Structs
 
-Unless you require functionality that can only be provided by a class (like identity or deinitializers), implement a struct instead.
+Unless you require functionality that can only be provided by a class, implement a struct instead.
 
+Additional capabilities of classes:
+
+- Inheritance: Enables one class to inherit the characteristics of another
+- Type casting: Enables you to check and interpret the type of a class instance at runtime
```

Comment:
It's true and I agree. 
But it would also point somewhere that you can achieve same goal with Composition. So people won't chose classes only because of inheritance, but also thought about structs with composition.